### DISCUSSION TITLE
v2ray configuration support yaml format

### DISCUSSION THREAD

**Discussion by xxm404** [v2fly/v2ray-core#2394] [Ideas]
v2ray-core now only supports json format configuration, maybe we can make v2ray support configuration in yaml format,yaml is usually more readable than json

**Comment by mydogshitgold** [comment]
Actually V2Ray supports YAML, TOML and protobuf binary for v4 configurations. You can use the `-format` flag to specify a configuration format and the command `v2ray convert -o=format -i=format` to convert between different formats. You can find more about it from help command: `v2ray help convert`.

Currently only JSON is supported for the new draft v5 configuration schema, but it would be easy to support other formats once the v5 schema is finalised.

**Reply by xxm404** [reply]
Thank you very much for your reply, I will try it.
